### DISCUSSION TITLE
Proxmox Update Repositories

### DISCUSSION THREAD

**Discussion by vismuthxx** [community-scripts/ProxmoxVE#602] [General]
the question is why the PVE 8.3 script does not work, I download the application and get this message, The process is complete. Repositories have been switched to community-scripts/ProxmoxVE. how do I stock up on a script?

**Comment by MickLesk** [comment]
What? Can you also formulate your question in such a way that people understand what you want to tell us or ask us?

Update Repository only does something to the LXC. It has nothing to do with the postinstall or anything else.

**Reply by vismuthxx** [reply]
Proxmox Update Repositories
this script bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/misc/update-repo.sh)" does not want to start displays a message
The process is complete. The repositories have been switched to community-scripts/ProxmoxVE

**Reply by MickLesk** [reply]
I don't understand your problem or your request. The script does exactly what it is supposed to do. I have tested it right now. It goes through all LXCs and updates the usr/bin/update file to the new repo.

**Reply by vismuthxx** [reply]
my script doesn't do anything.

**Reply by MickLesk** [reply]
Where you execute it?

**Reply by vismuthxx** [reply]
in proxmox ve 8.3.0 on hardware, I reinstalled proxmox. before that, everything was normally installed on 8.2.2

**Reply by MickLesk** [reply]
Do you have any idea what the script does? Apparently not. Read the website again.
